boards/arm/nrf9160_pca10090/nrf9160_pca10090_common.dts

```diff
@@ -121,6 +121,10 @@
 	status = "ok";
 };
 
+&timer1 {
```

Comment:
Why does it need to be enabled? Can't see from commit message :)

Reply:
So it can be selected by user for hardware byte counting. Other timers are enabled by default. Also in other socs all timers are enabled. Just following the pattern here